README.md

```diff
@@ -17,26 +17,23 @@ Vets-api requires:
 1. Install Postgres (on Mac): `brew install postgres`
 1. Get the code: `git clone https://github.com/department-of-veterans-affairs/vets-api.git; git submodule init; git submodule update`
```

Comment:
While you're editing this, may want to add that you need to cd into vets-api before calling git submodule